### ISSUE TITLE
Config schema violation: `hotkeySettings/19/action` must be equal to one of the allowed values

### ISSUE THREAD

**Issue by coolkiwiii** [VOICEVOX/voicevox#1178]
## 不具合の内容

When I cloned the repository and ran the command `npm run electron:serve`, I get an error that prevents it from showing the UI. It opens an Electron instance, but no UI is shown. I did not modify `background.ts`.

### 現象・ログ

```
INFO  Launching Electron...
App threw an error during load
Error: Config schema violation: 'hotkeySettings/19/action' must be equal to one of the allowed values
    at ElectronStore._validate (webpack://voicevox/./node_modules/conf/dist/source/index.js?:355:15)
    at get store [as store] (webpack://voicevox/./node_modules/conf/dist/source/index.js?:280:18)
    at new Conf (webpack://voicevox/./node_modules/conf/dist/source/index.js?:132:32)
    at new ElectronStore (webpack://voicevox/./node_modules/electron-store/index.js?:69:3)
    at eval (webpack://voicevox/./src/background.ts?:106:15)
    at ./src/background.ts (/Users/coolkiwiii/Documents/voicevox/dist_electron/index.js:4632:1)
    at __webpack_require__ (/Users/coolkiwiii/Documents/voicevox/dist_electron/index.js:6176:42)
    at /Users/coolkiwiii/Documents/voicevox/dist_electron/index.js:6243:37
    at Object.<anonymous> (/Users/coolkiwiii/Documents/voicevox/dist_electron/index.js:6245:12)
    at Module._compile (node:internal/modules/cjs/loader:1120:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1175:10)
    at Module.load (node:internal/modules/cjs/loader:988:32)
    at Module._load (node:internal/modules/cjs/loader:829:12)
    at c._load (node:electron/js2c/asar_bundle:5:13343)
    at loadApplicationPackage (/Users/coolkiwiii/Documents/voicevox/node_modules/electron/dist/Electron.app/Contents/Resources/default_app.asar/main.js:121:16)
    at Object.<anonymous> (/Users/coolkiwiii/Documents/voicevox/node_modules/electron/dist/Electron.app/Contents/Resources/default_app.asar/main.js:233:9)
[18:45:15.928] [error] Error: Config schema violation: 'hotkeySettings/19/action' must be equal to one of the allowed values
    at ElectronStore._validate (webpack://voicevox/./node_modules/conf/dist/source/index.js?:355:15)
    at get store [as store] (webpack://voicevox/./node_modules/conf/dist/source/index.js?:280:18)
    at new Conf (webpack://voicevox/./node_modules/conf/dist/source/index.js?:132:32)
    at new ElectronStore (webpack://voicevox/./node_modules/electron-store/index.js?:69:3)
    at eval (webpack://voicevox/./src/background.ts?:106:15)
    at ./src/background.ts (/Users/coolkiwiii/Documents/voicevox/dist_electron/index.js:4632:1)
    at __webpack_require__ (/Users/coolkiwiii/Documents/voicevox/dist_electron/index.js:6176:42)
    at /Users/coolkiwiii/Documents/voicevox/dist_electron/index.js:6243:37
    at Object.<anonymous> (/Users/coolkiwiii/Documents/voicevox/dist_electron/index.js:6245:12)
    at Module._compile (node:internal/modules/cjs/loader:1120:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1175:10)
    at Module.load (node:internal/modules/cjs/loader:988:32)
    at Module._load (node:internal/modules/cjs/loader:829:12)
    at c._load (node:electron/js2c/asar_bundle:5:13343)
    at loadApplicationPackage (/Users/coolkiwiii/Documents/voicevox/node_modules/electron/dist/Electron.app/Contents/Resources/default_app.asar/main.js:121:16)
    at Object.<anonymous> (/Users/coolkiwiii/Documents/voicevox/node_modules/electron/dist/Electron.app/Contents/Resources/default_app.asar/main.js:233:9)
objc[37999]: Class WebSwapCGLLayer is implemented in both /System/Library/Frameworks/WebKit.framework/Versions/A/Frameworks/WebCore.framework/Versions/A/Frameworks/libANGLE-shared.dylib (0x2392e7110) and /Users/coolkiwiii/Documents/voicevox/node_modules/electron/dist/Electron.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Libraries/libGLESv2.dylib (0x1190c1d08). One of the two will be used. Which one is undefined.
```

### 再現手順

Clone repository, run `npm ci`, run `npm run electron:serve`

### 期待動作

With no errors, it should show the VOICEVOX Editor UI.

## VOICEVOXのバージョン

0.14.2

## OSの種類/ディストリ/バージョン

- [ ] Windows
- [x] macOS 13.1 Ventura (Apple M1 Processor)
- [ ] Linux

**Comment by sevenc-nanashi** [comment]
I think your config.json is broken. Try deleting `~/Library/Application Support/voicevox-dev/config.json` or renaming to other name, like `config.bak.json`.

**Comment by Hiroshiba** [comment]
I'm wondering what `config.json` caused the error. If possible, please post the `config.json` here...!

**Comment by Hiroshiba** [comment]
`config.json`のパースエラーが起きたとき、どういう処理をすればいいかの開発者向け案内を出しても良いかもですね。

**Comment by coolkiwiii** [comment]
Yes, it was the `config.json` file! It must have been a leftover file from last year when I cloned the repository then. Thank you so much! Here is the old `config.json` file, it seems to be from May 2022. (GitHub wouldn't allow me to upload just the .json file, so I had to compress it into a .zip archive)
[config_old.json.zip](https://github.com/VOICEVOX/voicevox/files/10610325/config_old.json.zip)
